gpflow/kernels/base.py

```diff
@@ -98,7 +98,6 @@ def slice(self, X: tf.Tensor, X2: Optional[tf.Tensor] = None):
             if X2 is not None:
                 X2 = X2[..., dims]
         elif dims is not None:
-            # TODO(@awav): Convert when TF2.0 will support proper slicing.
```

Comment:
is there any tensorflow issue that we could link here if they enable proper slicing at some point ? how likely is that to happen?

Reply:
Tens of issues and 100% not going to happen.